src/explore-education-statistics-admin/src/styles/patterns/editor.scss

```diff
@@ -3,42 +3,62 @@ div.ck.ck-editor__editable_inline {
   background-color: #eee;
 }
 
-.editable-content {
+.page-editing .editable-content {
```

Comment:
Shouldn't this stylesheet be a CSS module instead of global stylesheet?

Reply:
Moved the Editable content classes into a module and updated it to use them.